docs/it/token_endpoint.rst

```diff
@@ -224,6 +224,11 @@ ID Token
 ++++++++
 
 L'ID Token è un JSON Web Token (JWT) che contiene informazioni sull'utente che ha eseguito l'autenticazione. I RP DEVONO eseguire la validazione dell'ID Token.
+
```

Comment:
Solo per SPID: se sono richiesti attributi utente tramite il parametro scope o tramite il parametro claims, il metadata del RP DOVREBBE contenere id_token_encrypted_response_alg e id_token_encrypted_response_enc ed il loro valore dovrebbe essere diverso da 'none'. Se sono richiesti attributi utente tramite il parametro scope o tramite il parametro claims ed il metadata del RP non contiene id_token_encrypted_response_alg e id_token_encrypted_response_enc oppure il loro valore è 'none', nessun attributo dell'utente sarà presente nell'id_token restituito dall'endpoint token.

Reply:
here my proposal

https://github.com/italia/spid-cie-oidc-docs/issues/82

so if you agree we may merge it as it is and then create a separate PR for that